boards/arm/stm32h747i_disco/doc/index.rst

```diff
@@ -63,6 +63,15 @@ The current Zephyr stm32h747i_disco board configuration supports the following h
 +-----------+------------+-------------------------------------+
 | GPIO      | on-chip    | gpio                                |
 +-----------+------------+-------------------------------------+
+| ETHERNET  | on-chip    | ethernet  (*)                       |
++-----------+------------+-------------------------------------+
+| RNG       | on-chip    | True Random number generator        |
++-----------+------------+-------------------------------------+
+
+(*) From UM2411 Rev 4:
+   With the default setting, the Ethernet feature is not working because of
+   a conflict between ETH_MDC and SAI4_D1 of the MEMs digital microphone.
+   Make sure you have SB8 closed and SB21 open to get Ethernet working.
```

Comment:
As I remember, modifying SB78 and SB21 is necessary, but not sufficient:
* SB21, SB45 and R87 should be opened
* SB22, SB44, SB17 and SB8 should be closed

see: https://os.mbed.com/teams/ST/wiki/DISCO_H747I-modifications-for-Ethernet
Could be interesting to point to this link as there are picture to help finding the required changes.

Reply:
Would be nice if Zephyr docs can provide its own set of pictures.

Reply:
Please note that Pictures are mine :smiley: 
And so, those pictures can be reused for zephyr docs.

Reply:
@ABOSTM Great, copyright clearance then :smile:

Reply:
If @ABOSTM agrees, would be it okay, to put [this URL](https://os.mbed.com/teams/ST/wiki/DISCO_H747I-modifications-for-Ethernet) with very short description to appropriate index.rst file?

Reply:
I vote for making these images part of Zephyr docs.

Reply:
I agree that pictures would be required (and available directly rather than a link)

Reply:
Images and corresponding description were added.